physionet-django/training/models.py

```diff
@@ -38,14 +36,12 @@ class Meta:
             ("can_view_course_guidelines", "Can view course guidelines"),
         ]
 
-    def expire_course_version(self, instance, number_of_days):
+    def archive_course_version(self):
         """
         This method expires the course by setting the is_active field to False and expires all
```

Comment:
Change expires to archives in the docstring? 

I think "expires" should just be used when referring to training that has reached the expiry date (i.e. if someone takes a course that is valid for one year, the course expires when a year has elapsed).